js/router.js

```diff
@@ -83,6 +86,8 @@ export default class ObRouter extends Router {
     if (tab === 'channel') {
       pageOpts.category = args[0];
       pageOpts.layer = args[1];
+    } else {
+      pageOpts.action = args[0];
```

Comment:
This will be used in the future for actions like customizing the page that are triggered from a nav link.